cql2/standard/clause_7_enhanced.adoc

```diff
@@ -112,14 +112,62 @@ category NOT IN (1,2,3,4)
 ----
 ====
 
-[[case-insensitive-comparison]]
-=== Requirements Class "Case-insensitive Comparison"
+[[accent-case-insensitive-comparison]]
+=== Requirements Class "Accent and Case-insensitive Comparison"
 
-include::requirements/requirements_class_case-insensitive-comparison.adoc[]
+include::requirements/requirements_class_accent-case-insensitive-comparison.adoc[]
 
-This requirements class adds support for case-insensitive string comparisons.
+The following requirements class adds support for case-insensitive string comparisons.
 
-#TODO: add requirement for UPPER() and LOWER(). However, there is still the unresolved question how we exactly define UPPER() and LOWER(). Or whether we should define a unicode case folding function instead (see http://unicode.org/reports/tr21/tr21-5.html#Caseless_Matching[Unicode] or https://www.w3.org/TR/charmod-norm/#definitionCaseFolding[W3C]).#
+include::requirements/accent-case-insensitive-comparison/REQ_casei-builtin-function.adoc[]
+
+[[example_7_4_casei]]
+.Example case-insensitive comparison
+====
+
+----
+CASEI(road_class) IN (CASEI('Οδος'),CASEI('Straße'))
+----
+
+[source,JSON]
+----
+{
+  "in": [
+    { "function": { "name": "casei", "arguments": ["road_class"] } },
+    [
+      { "function": { "name": "casei", "arguments": ["Οδος"] } },
+      { "function": { "name": "casei", "arguments": ["Straße"] } }
+    ]
+  ]
+}
+----
+====
+
+This requirements class adds support for accent-insensitive string comparisons.
+
+include::requirements/accent-case-insensitive-comparison/REQ_accenti-builtin-function.adoc[]
+
+[[example_7_4_accenti]]
+.Example accent-insensitive comparison
+====
+
+----
+etat_vol = ACCENTI('débárquér')
```

Comment:
@pvretano I know you just merged this, but shouldn't this be 

```
ACCENTI(etat_vol) = ACCENTI('débárquér')
```

as as it is for CASEI? same for the JSON example.